File: Pcote_MeusPJ/C/calcul.c
#include <stdio.h>

int main() {
    char operador;
    double num1, num2;

    printf("Digite o operador (+, -, *, /): ");
    scanf("%c", &operador);

    printf("Digite dois números: ");
    scanf("%lf %lf", &num1, &num2);

    switch(operador) {
        case '+':
            printf("%.2lf + %.2lf = %.2lf", num1, num2, num1 + num2);
            break;
        case '-':
            printf("%.2lf - %.2lf = %.2lf", num1, num2, num1 - num2);
            break;
        case '*':
            printf("%.2lf * %.2lf = %.2lf", num1, num2, num1 * num2);
            break;
        case '/':
            if(num2 != 0)
                printf("%.2lf / %.2lf = %.2lf", num1, num2, num1 / num2);
            else
                printf("Erro! Divisão por zero!");
            break;
        default:
            printf("Operador inválido!");
    }

    return 0;
}

File: Pcote_MeusPJ/Python/Calcul.py
def soma(a, b):
    return a + b

def subtracao(a, b):
    return a - b

def multiplicacao(a, b):
    return a * b

def divisao(a, b):
    if b != 0:
        return a / b
    else:
        return "Erro: divisão por zero"

# Exibição das operações disponíveis
print("Operações disponíveis:")
print("+ para soma")
print("- para subtração")
print("* para multiplicação")
print("/ para divisão")

# Entrada dos números e operador
operador = input("Digite o operador (+, -, *, /): ")
num1 = float(input("Digite o primeiro número: "))
num2 = float(input("Digite o segundo número: "))

# Realiza a operação e imprime o resultado
if operador == "+":
    resultado = soma(num1, num2)
elif operador == "-":
    resultado = subtracao(num1, num2)
elif operador == "*":
    resultado = multiplicacao(num1, num2)
elif operador == "/":
    resultado = divisao(num1, num2)
else:
    resultado = "Operador inválido"

print("Resultado:", resultado)
